guides/v2.2/comp-mgr/upgrader/upgrade.md

```diff
@@ -37,15 +37,25 @@ bin/magento cache:clean
 
 After the upgrade completes, manually clear `var` subdirectories:
 
-	rm -rf <Magento install dir>/var/cache/*
-	rm -rf <Magento install dir>/var/page_cache/*
-	rm -rf <Magento install dir>/generated/code/*
+```bash
```

Comment:
You should be able to delete the contents of the three directories in one command.